app/src/app/close.js

```diff
@@ -49,8 +52,16 @@ export default function(e) {
     }
     if (isOSX()) app.hide()
   }
+  const lastUrl = state.mainWindow.webContents.getURL()
+  const isLastUrlChat = (
+    parseUrl(lastUrl)
+      .pathname
+      .split('/')
+      .filter(p => p !== '')[0]
+  ) === chatPath
+
   storage.set('lastUrl', {
-    url: state.mainWindow.webContents.getURL(),
+    url: isLastUrlChat ? lastUrl : false,
```

Comment:
I set it to `null` originally but that doesn't work on windows. When it is `null` the storage file `lastUrl.json` is empty and this is a problem later when electron-json-storage tries to load `lastUrl` because it generates a generic error. And it goes [here](https://github.com/ubergrape/grape-electron/blob/master/app/src/app/initApp.js#L85) which gives a blank page to the users.